collective/searchandreplace/searchreplaceutility.py

```diff
@@ -426,7 +436,7 @@ def getRawTextField(obj, field):
         if isinstance(baseunit, tuple):
             #  LinesField
             text = "\n".join(baseunit)
-        elif isinstance(baseunit.raw, unicode):
+        elif hasattr(baseunit, 'raw') and isinstance(baseunit.raw, unicode):
```

Comment:
`unicode` does not exist on Python 3, so it would give a `NameError`. I guess this code path is not tested. I see this was already the case before your PR, but could you fix it here?

Reply:
This would not be executed on Py3 as we have no Archetypes.
But I'll change it anyway.